tool/tctl/common/auth_command.go

```diff
@@ -748,6 +754,53 @@ func (a *AuthCommand) checkProxyAddr(clusterAPI auth.ClientI) error {
 	return trace.BadParameter("couldn't find registered public proxies, specify --proxy when using --format=%q", identityfile.FormatKubernetes)
 }
 
+func (a *AuthCommand) logUngrantedTTL(clusterAPI auth.ClientI) error {
+	rc := clockwork.NewRealClock()
```

Comment:
A local-variable real clock doesn't have much of a reason for existing, ditch for time.Now()?